From 45b16a9b2782610ffa3507b1636a4a4651631246 Mon Sep 17 00:00:00 2001
From: Shengjiu Wang <shengjiu.wang@nxp.com>
Date: Tue, 23 Jan 2018 13:27:10 +0800
Subject: [PATCH 3240/5242] MLK-16224-2: ASoC: dmaengine_pcm: add fifo_num to
 snd_dmaengine_dai_dma_data

commit  34db4e7440d211e67bf5fbfa71ab701a062c616e from
https://source.codeaurora.org/external/imx/linux-imx.git

In order to support multi-fifo sdma script, the audio driver need to send
the fifo number to dma driver through dma_slave_config, and the cpu_dai
driver should config fifo_num for the audio platform driver, then platform
driver can config fifo_num to dma.
So add new variable fifo_num for struct snd_dmaengine_dai_dma_data.

Signed-off-by: Shengjiu Wang <shengjiu.wang@nxp.com>
Reviewed-by: Robin Gong<yibin.gong@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 include/sound/dmaengine_pcm.h |    1 +
 1 file changed, 1 insertion(+)

diff --git a/include/sound/dmaengine_pcm.h b/include/sound/dmaengine_pcm.h
index 800cf27..a55b1ae 100644
--- a/include/sound/dmaengine_pcm.h
+++ b/include/sound/dmaengine_pcm.h
@@ -86,6 +86,7 @@ struct snd_dmaengine_dai_dma_data {
 	const char *chan_name;
 	unsigned int fifo_size;
 	unsigned int flags;
+	unsigned int fifo_num;
 	bool (*check_xrun)(struct snd_pcm_substream *substream);
 	void (*device_reset)(struct snd_pcm_substream *substream, bool stop);
 };
-- 
1.7.9.5

